daprdocs/content/en/contributing/docs-contrib/contributing-docs.md

```diff
@@ -1,82 +1,86 @@
+- Name your PNG files using the convention of a concept or building block so that they are grouped.
+  - For example: `service-invocation-overview.png`. 
+  - For more information on calling out images using shortcode, see the [Images guidance](#images) section below.
+- Add the diagram to the correct section in the `Dapr-Diagrams.pptx` deck so that they can be amended and updated during routine refresh.
 
 ## Contributing a new docs page
-- Make sure the documentation you are writing is in the correct place in the hierarchy.
-- Avoid creating new sections where possible, there is a good chance a proper place in the docs hierarchy already exists.
-- Make sure to include a complete [Hugo front-matter](#front-matter).
-
-### Contributing a new concept doc
-- Ensure the reader can understand why they should care about this feature. What problems does it help them solve?
-- Ensure the doc references the spec for examples of using the API.
-- Ensure the spec is consistent with concept in terms of names, parameters and terminology. Update both the concept and the spec as needed.
-- Avoid just repeating the spec. The idea is to give the reader more information and background on the capability so that they can try this out. Hence provide more information and implementation details where possible.
-- Provide a link to the spec in the [Reference]({{<ref reference >}}) section.
-- Where possible reference a practical How-To doc.
-
-### Contributing a new How-To guide
-- `How To` articles are meant to provide step-by-step practical guidance on to readers who wish to enable a feature, integrate a technology or use Dapr in a specific scenario.
-- Sub directory naming - the directory name should be descriptive and if referring to specific component or concept should begin with the relevant name. Example *pubsub-namespaces*.
-- Do not assume the reader is using a specific environment unless the article itself is specific to an environment. This include OS (Windows/Linux/MacOS), deployment target (Kubernetes, IoT etc.) or programming language. If instructions vary between operating systems, provide guidance for all.
-- Include code/sample/config snippets that can be easily copied and pasted.
-- At the end of the article, provide the reader with related links and next steps (this can be other relevant "how-to", samples for reference or related concepts).
+
+If you're creating a new article, make sure you:
+
+- Place the new document in the correct place in the hierarchy.
+- Avoid creating new sections. Most likely, the proper place is already in the docs hierarchy.
+- Include complete [Hugo front-matter](#front-matter).
+
+Select the topic type below to view a suggested template to help you get started.
+
+| Topic type | What is it? |
+| ---------- | ----------- |
+| [Concept]({{< ref "docs-templates/concept-template.md" >}}) | Answers the question, "What problems does this help me solve?" Avoid repeating the API or component spec; provide more details. |
```

Comment:
This is causing the builds to fail. This page can't be found

Reply:
Also with `ref` shortcodes you want to link just to `concept-template.md`, not `docs-templates/concept-template.md`. `ref` expects a globally unique file name, not a path